client/spec.yaml

```diff
@@ -495,4 +495,178 @@ paths:
           description: Bad request
         "5XX":
           description: Server side error while tailing logs
-
+  /api/logs/v1/{tenant}/loki/api/v1/rules:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Get Logs Rules for a tenant in JSON form
+      operationId: getAllLogsRules
+      description: |
+        You can get Logs rules for a tenant with health data
+      responses:
+        "2XX":
+          description: rule for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: "./responses/responses.yaml#/components/schemas/AllLogRulesResponse"
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/loki/api/v1/rules/{namespace}:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+      - $ref: ./parameters/parameters.yaml#/components/parameters/logRulesNamespace
```

Comment:
I'm not sure how `$ref` works but does `logRulesNamespace` need to be named as `namespace` so that it can be replaced in `/api/logs/v1/{tenant}/loki/api/v1/rules/{namespace}`? or naming does not matter and it just refers sequentially to the params in the path (in this case, `tenant` -> `tenant`, `namespace` -> `logRulesNamespace`?

Reply:
@jessicalins AFAIK the path name `namespace` needs to map into the parameter's name definition `logRulesNamespace.name`, i.e.:
```
    logRulesNamespace:
      in: path
      name: namespace
      description: namespace of the log rule
      required: true
      schema:
        type: string
```

Reply:
ah I see! makes sense :+1: